configs/vico/README.md

```diff
@@ -0,0 +1,105 @@
+# ViCo (2023)
+
+> [ViCo: Detail-Preserving Visual Condition for Personalized Text-to-Image Generation](https://arxiv.org/abs/2208.12242)
+
+> **Task**: Text2Image
+
+<!-- [ALGORITHM] -->
+
+## Abstract
+
+<!-- [ABSTRACT] -->
+
+Personalized text-to-image generation using diffusion models has recently been proposed and attracted lots of attention. Given a handful of images containing a novel concept (e.g., a unique toy), we aim to tune the generative model to capture fine visual details of the novel concept and generate photorealistic images following a text condition. We present a plug-in method, named ViCo, for fast and lightweight personalized generation. Specifically, we propose an image attention module to condition the diffusion process on the patch-wise visual semantics. We introduce an attention-based object mask that comes almost at no cost from the attention module. In addition, we design a simple regularization based on the intrinsic properties of text-image attention maps to alleviate the common overfitting degradation. Unlike many existing models, our method does not finetune any parameters of the original diffusion model. This allows more flexible and transferable model deployment. With only light parameter training (~6% of the diffusion U-Net), our method achieves comparable or even better performance than all state-of-the-art models both qualitatively and quantitatively.
+
+<!-- [IMAGE] -->
+
+<div align=center>
+<img src="https://github.com/haoosz/ViCo/assets/71176040/0ee95a57-fecf-4bba-bc64-eda46e5cc6d1">
+</div>
+
+## Configs
+
+|                                     Model                                      | Dataset | Download |
+| :----------------------------------------------------------------------------: | :-----: | :------: |
+|                         [ViCo](./vico.py)                          |    -    |    -     |
+
+## Pretrained Models
+To Do
+## Quick Start
+
+1. Download [data](https://drive.google.com/drive/folders/1m8TCsY-C1tIOflHtWnFzTbw2C6dq67mC) and save to `data/vico/`
+
+The file structure will be like this:
+
+```text
+data
+└── vico
+    └──batman
+       ├── 1.jpg
+       ├── 2.jpg
+       ├── 3.jpg
+       └── 4.jpg
+    └──clock
+       ├── 1.jpg
+       ├── 2.jpg
+       ├── 3.jpg
+       └── 4.jpg
+    ...
+```
+
+2. Start training with the following command:
+
+```bash
+bash tools/dist_train.sh configs/vico/vico.py 1
+```
+
+<!-- <table align="center">
+<thead>
+  <tr>
+    <td>
+<div align="center">
+  <img src="https://user-images.githubusercontent.com/28132635/232682088-26424e69-f697-49bc-a706-d03245ff25b1.png" width="400"/>
+  <br/>
+  <b>'dreambooth'</b>
+</div></td>
+    <td>
+<div align="center">
+  <img src="https://user-images.githubusercontent.com/28132635/232682057-fbc99047-e2d0-433e-bbc5-4f2d4ec18191.png" width="400"/>
+  <br/>
+  <b>'dreambooth-lora'</b>
+</div></td>
+    <td>
+</thead>
+</table> -->
+
```

Comment:
You need to provide quick start instructions for training, testing and inference.

Reply:
[instructions](https://github.com/open-mmlab/mmagic/pull/1978/files#diff-40d41ed519c7e44c2f8d680b7713273c027fb56d06a89109e7fa5e16cd298420R27-R116) are provided.